lib/views/play/widgets/board_square.dart

```diff
@@ -29,13 +29,14 @@ class BoardSquare extends ConsumerWidget {
 
         if (cell.hasMine) {
           ref.read(cellNotifierProvider(pos).notifier).setOpen();
-          ref.read(logNotifierProvider.notifier).add(pos);
         } else {
           ref.read(cellNotifierProvider(pos).notifier).open();
         }
+        ref.read(logNotifierProvider.notifier).add(pos);
```

Comment:
I think can remove duplicate `ref.read(logNotifierProvider.notifier)`

Reply:
which duplicates do you mean?

Reply:
I menat line 35 and line 39

Reply:
got it. i'll consider this in the next refactoring.